src/index.tsx

```diff
@@ -7,14 +7,19 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { App } from "./Components/App/App";
 import * as serviceWorker from "./serviceWorker";
-
+import { FeatureToggleClient } from "FeatureToggleClient";
+import { AuthorizationClient } from "@itwinjs-sandbox";
```

Comment:
Because of imports in AuthorizationClient, it drags in complete @bentley/itwin-client and some other deps to initial app bundle, causing it to bloat again to 5.5Mb

Reply:
Removed imports